test/system/250-systemd.bats

```diff
@@ -280,6 +280,12 @@ LISTEN_FDNAMES=listen_fdnames" | sort)
     is "${container_uuid}" "${output:0:32}" "UUID should be first 32 chars of Container id"
 }
 
+@test "podman --systemd fails on cgroup v1 with a private cgroupns" {
+    skip_if_cgroupsv2
+
+    run_podman 126 run --systemd=always --cgroupns=private $IMAGE true
```

Comment:
Please add `assert "$output" =~ "blah blah blah blah"`. 126 can happen for many reasons, it is important to verify that it's happening here for the expected reason.